frappe/website/render.py

```diff
@@ -125,12 +125,12 @@ def build_response(path, data, http_status_code, headers=None):
 	response = Response()
 	response.data = set_content_type(response, data, path)
 	response.status_code = http_status_code
-	response.headers["X-Page-Name"] = path.encode("utf-8")
+	response.headers["X-Page-Name"] = path.encode("ascii", errors="xmlcharrefreplace")
```

Comment:
These changes are already merged. Please rebase.

Reply:
This has been added by mistake and I will remove it in a rebase as suggested...